src/extract_flags_from_node_args.js

```diff
@@ -0,0 +1,7 @@
+export default function extractFlagsFromNodeArgs( nodeArgs ) {
```

Comment:
```suggestion
/**
 * This is a command line flag reader for runTest.js.
 * The command line argument order and parameter are fixed (see call to runTest in index.js),
 *  so we can assume the correct position and value of all parameters.
 */
export default function extractFlagsFromNodeArgs( nodeArgs ) {
```
I was a bit confused by this file and had to look up its usage and why we can assume fixed command line param positions.